Sources/SQLKit/Builders/Implementations/SQLInsertBuilder.swift

```diff
@@ -0,0 +1,384 @@
+/// Builds ``SQLInsert`` queries.
+///
+/// > Note: Although in the strictest sense, this builder could conform to ``SQLUnqualifiedColumnListBuilder``, doing
+/// > so would be semantically inappropriate. the protocol documents its `columns()` methods as being additive, but
+/// > ``SQLInsertBuilder``'s otherwise-identical public APIs overwrite the effects of any previous invocation. It
+/// > would ideally be preferable to change ``SQLInsertBuilder``'s semantics in this regard, but this would be a
+/// > significant breaking change in the API's behavior, and must therefore wait for a major version bump.
+public final class SQLInsertBuilder: SQLQueryBuilder, SQLReturningBuilder/*, SQLUnqualifiedColumnListBuilder*/ {
+    /// The ``SQLInsert`` query this builder builds.
+    public var insert: SQLInsert
+    
+    // See `SQLQueryBuilder.database`.
+    public var database: any SQLDatabase
+    
+    // See `SQLQueryBuilder.query`.
+    @inlinable
+    public var query: any SQLExpression {
+        self.insert
+    }
+
+    // See `SQLReturningBuilder.returning`.
+    @inlinable
+    public var returning: SQLReturning? {
+        get { self.insert.returning }
+        set { self.insert.returning = newValue }
+    }
+    
+    /// Creates a new `SQLInsertBuilder`.
+    @inlinable
+    public init(_ insert: SQLInsert, on database: any SQLDatabase) {
+        self.insert = insert
+        self.database = database
+    }
+    
+    /// Use an `Encodable` value to generate a row to insert and add that row to the query.
+    ///
+    /// Example usage:
+    ///
+    /// ```swift
+    /// let earth = Planet(id: nil, name: "Earth", isInhabited: true)
+    ///
+    /// try await sqlDatabase.insert(into: "planets")
+    ///     .model(earth, keyEncodingStrategy: .convertToSnakeCase)
+    ///     .run()
+    ///
+    /// // Effectively the same as:
+    /// try await sqlDatabase.insert(into: "planets")
+    ///     .columns("id", "name", "is_inhabited")
+    ///     .values(SQLBind(earth.id), SQLBind(earth.name), SQLBind(earth.isInhabited))
+    ///     .run()
+    /// ```
+    ///
+    /// > Note: The term "model" does _not_ refer to Fluent's `Model` type. Fluent models are not compatible with
```

Comment:
IMO this isn't needed, this package can be used without Fluent too so people who might do that could be confused

Reply:
In this version of the package, Fluent is tied to its usage in practically all use cases, so I think the warning is appropriate. However, this will not be the case after this point, so these can be removed in the major version update.